app/e2e/outgoingcall.test.js

```diff
@@ -0,0 +1,82 @@
+import { by, element, expect, waitFor, device } from 'detox';
+
+describe('Outgoing Call', () => {
```

Comment:
Let's continue the discussion here regarding the following test case, so we can separate the thread

>> - check if call is disconnected after callee disconnects the call or times out

> Can setup another number and use Twilio verb to end

I don't think we need to. We can just wait until the current call times out? It's only 10s long right?